packages/loader/__tests__/index-test.ts

```diff
@@ -0,0 +1,104 @@
+import * as fs from 'fs';
+import compiler from './compiler';
+import mutationSchema from '../../../__tests__/shared/mutationSchema';
+import querySchema from '../../../__tests__/shared/simpleSchema';
+
+describe('Loader', () => {
+  let originalTimeout: number;
+  beforeEach(() => {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
+  });
+
+  afterEach(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+  });
+
+  describe('Mutation', () => {
+    [
+      'mutationNoArgs',
```

Comment:
All of these tests may be overkill, but wanted to add some in to test different situations. Same with the 'Query' tests.

Reply:
@BrandonHermanAvant I agree that these are overkill. I think we should just test that the things are passed to `@gql2ts/from-query` which is already tested pretty well 😄